### DISCUSSION TITLE
Allow stop/down stack without removing stack itself

### DISCUSSION THREAD

**Discussion by ToshY** [louislam/dockge#51] [Feature Request]
**Problem**

Currently there's only an option to `delete` the stack, which performs a `docker compose down` and a complete removal of the stack directory.

**Request**

Add "stop" button/option that just performs a `docker compose down`.

**Comment by mitchplze** [comment]
Maybe only recently added, but this works well for me:

<img width="1095" alt="image" src="https://github.com/louislam/dockge/assets/25162574/97999ed1-ea69-4015-aef8-1f723acd3107">

**Comment by MSalim7** [comment]
When the stop button is pressed, the container is stopped and the container is not deleted. So it's not docker compose down. Docker compose down deletes the container. Please add another "**Down**" button to the author of the project

**Reply by mitchplze** [reply]
Good catch. Agreed, this would be good to implement.

**Comment by spupuz** [comment]
i would love it too

**Comment by nickshanks347** [comment]
This would be a great feature

**Comment by micmejia** [comment]
or put a toggle beside STOP button, e.g. "auto-delete containers on stop".

**Reply by louislam** [reply]
This is implemented.

![image](https://github.com/louislam/dockge/assets/1336778/ad9d2ca6-e2e7-4701-8187-955de04d10d9)

**Comment by FunctionDJ** [comment]
![image](https://github.com/louislam/dockge/assets/34871211/518a73f8-2b43-47e1-a0e3-e6a32eedfda0)
This is hella confusing. "Inactive" is not from any Docker terminology AFAIK. So is "Stop" = "docker compose stop" and "Stop & Inactive" = "docker compose down"?
